tests/test_mle.py

```diff
@@ -199,28 +200,20 @@ def test_infer_node_dgs_w_error_invariant_wrt_edge_order():
 
         seed = np.random.randint(1000)
 
-        dg_1, dg_err_1 = infer_node_vals_and_errs(
-            edge_idxs,
-            obs_edge_diffs,
-            edge_stddevs,
-            ref_node_idxs,
-            ref_node_vals,
-            ref_node_stddevs,
+        f = partial(
+            infer_node_vals_and_errs,
+            ref_node_idxs=ref_node_idxs,
+            ref_node_vals=ref_node_vals,
+            ref_node_stddevs=ref_node_stddevs,
+            n_bootstrap=10,
             seed=seed,
         )
 
+        dg_1, dg_err_1 = f(edge_idxs, obs_edge_diffs, edge_stddevs)
+
         p = np.random.permutation(len(edge_idxs))
         p = np.arange(len(edge_idxs))
```

Comment:
Surprise overwrite: Parameterize this test to accept permutation, then call test with random and identity permutations `p`?

Reply:
Good catch; this has been broken since its introduction. Fixed in https://github.com/proteneer/timemachine/pull/1020/commits/815b8bc30a6c4b77420c6f8c8c2285c804aad15c